integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/base/TestDataTypeJaxrs.java

```diff
@@ -255,4 +252,19 @@ public void testDefault_rt() {
     Integer result = restTemplate.getForObject(urlPrefix, Integer.class);
     assertEquals((int) result, 100);
   }
+
+  @Test
+  public void spaceCharEncode_intf() {
```

Comment:
this file will be deleted in next PR
and seems your new test case is not about datatype, it's about codec of path and query, create a new schema for it?

Reply:
The tests have been moved into `ParamCodecSchema` and `TestParamCodec`. Please review